a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -71,15 +71,15 @@ def execute(self, loadbalancers_list, amphora, member_list):
         for loadbalancer in loadbalancers_list:
             for pool in loadbalancer.pools:
                 for member in pool.members:
-                    if member.subnet_id:
+                    if member.subnet_id and member in member_list:
```

Comment:
In get_delete_pool_flow(), it will also use a10_network_tasks.CalculateDelta task but didn't provide the member_list.
So, if we add this condition. Will instances lost some interfaces when delete pool? **(in case of muliple lbs in project and other lb pool has members**)

Reply:
Added MEMBER_LIST to CalculateDelta task in delete_pool_flow().